tslint.json

```diff
@@ -118,7 +118,8 @@
     "missing-rollup-globals": [
       true,
       "./tools/package-tools/rollup-globals.ts",
-      "src/+(lib|cdk|material-examples|material-experimental|cdk-experimental)/**/*.ts"
-    ]
+      "src/+(lib|cdk|material-examples)/**/*.ts"
```

Comment:
Did you mean to remove `material-experimental|cdk-experimental` here?

Reply:
Absolutely not. This was an oversight.